.eslintrc.js

```diff
@@ -45,9 +55,26 @@ module.exports = {
         node: true
       },
       plugins: ['node'],
+      // This is a workaround for the fact that glob overrides are not allowed
+      // to use `extends` (https://github.com/eslint/eslint/issues/8813).
       rules: Object.assign({}, require('eslint-plugin-node').configs.recommended.rules, {
```

Comment:
FWIW, this is allowed now (I think I mentioned in my other review)

You can tweak this to:

```js
{
  // ...snip...
  extends: ['plugin:node/recommended'],
  rules: {
    // the actual overrides you need
  }
}
```

Reply:
Integrated this, plus our discussion in Discord DMs about the TS config – I like that and I think we'll probably upstream it into the ember-cli-typescript docs (or even blueprint?) for this that I want to supply.